src/plugin_discovery/plugin_pack/lib/fs.js

```diff
@@ -0,0 +1,58 @@
+import { stat, readdir } from 'fs';
+import { resolve } from 'path';
+
+import { fromNode as fcb } from 'bluebird';
+import { Observable } from 'rxjs';
+
+import { createInvalidDirectoryError } from '../../errors';
+
+async function statTest(path, test) {
+  try {
+    const stats = await fcb(cb => stat(path, cb));
```

Comment:
Doesn't it bug you when the `stat` function is imported "globally" into the module but unusable? I really like the `fcb` pattern because it allows me to use the actual function rather than aliasing it to `xyzAsync` in every file. https://github.com/spalger/fcb

Reply:
One reason I like `promisify` is that it is the pattern Node went for in core (with `util.promisify`). Also, re every file: You either do `fcb(...)` or `promisify(...)` in every file, so I don't feel that's a difference. I'm fine with keeping it, though.

Reply:
It's the aliasing that's different, not the import/using in every file. Though, in other projects, I've created adapters for libs like `fs` so that they are promisified in one place and reused all over which has been really nice, except that it means you're using a non-standard lib for something so basic as file access.

Reply:
Let's keep it as-is :+1: